.github/workflows/cla.yml

```diff
@@ -0,0 +1,22 @@
+name: "CLA Signature Bot"
+on:
+  issue_comment:
+    types: [created]
+  pull_request_target:
+    types: [opened,closed,synchronize]
+
+jobs:
+  CLABot:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Signature Bot"
+        uses: MetaMask/cla-signature-bot@v3.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          path-to-signatures: 'cla.json'
+          url-to-cladocument: 'https://metamask.io/cla.html'
+          # This branch can't have protections, commits are made directly to the specified branch.
+          branch: 'cla-signatures'
+          whitelist: dependabot
```

Comment:
Is this hardcoded? We should move to `safelist`/`allowlist` if possible.

Reply:
It's a custom property used by the action. We can certainly update our fork to use a different property name, sure.

Reply:
This has been resolved here: https://github.com/MetaMask/cla-signature-bot/commit/6adffa525467ef481332325d9cc1dbbb126a3436
And the new `allowlist` parameter is in-use as of 485431ce2fa6d65b43fbb1894f988ee4cc2d63f5